source/includes/connection-snippets/scram-snippets/php-connection.php

```diff
@@ -0,0 +1,10 @@
+<!-- begin scram connection -->
+<?php
+
+$client = new MongoDB\Client(
+    'mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority'
+);
+
+$db = $client->test;
```

Comment:
Do these examples acknowledge the fact that [driver client objects do no IO in their constructor](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#clients-do-no-i-o-in-the-constructor)? In most of these examples, the only IO performed will be the initial DNS lookup for the SRV/TXT record. Selecting a database or collection object from the client is no enough to initiate SDAM and perform an auth handshake with the server. This means that users could run many of these examples and SCRAM would not actually be exercised at all.

An easy, universal way to ensure the SCRAM handshake _would_ happen may be to execute a `ping` command using the newly constructed clients, or perhaps insert a test document.

Reply:
@mongomoe poke, in case you missed this comment.